pkg/apis/autoscaling/v1beta1/machineautoscaler_types.go

```diff
@@ -10,15 +10,21 @@ func init() {
 
 // MachineAutoscalerSpec defines the desired state of MachineAutoscaler
 type MachineAutoscalerSpec struct {
+	// MinReplicas constrains the minimal number of replicas
 	// +kubebuilder:validation:Minimum=0
 	MinReplicas int32 `json:"minReplicas"`
+
+	// MaxReplicas constrains the maximal number of replicas
 	// +kubebuilder:validation:Minimum=1
-	MaxReplicas    int32                       `json:"maxReplicas"`
+	MaxReplicas int32 `json:"maxReplicas"`
```

Comment:
can we please add a little more details like the suggesttion in above comment.

Reply:
done